assets/src/scss/layout/navbar/_search.scss

```diff
@@ -167,40 +167,48 @@
   }
 }
 
-.nav-search-toggle-container {
+.nav-search-toggle {
+  background: transparent;
+  border: none;
+  border-radius: 50%;
+  height: 44px;
+  width: 44px;
   display: none;
-  padding: $sp-1 $sp-1x;
+  padding: $sp-1 $sp-1;
+  margin: auto $sp-1;
+  position: relative;
+
+  &:focus {
+    @include focus-styles(50%);
+  }
+
+  &.open --top-navigation--search-toggle-- {
+    background: var(--grey-200);
+  }
 
   &.medium-and-less {
     display: inline-block;
+
     @include large-and-up {
       display: none;
     }
   }
 
   @include large-and-up {
-    border-inline-start: var(--top-navigation--separation);
     display: inline-block;
-    padding: $sp-1x;
+
+    &:before {
+      content: "";
+      position: absolute;
+      height: 100%;
+      width: 1px;
+      top: 0;
+      left: -$sp-1;
```

Comment:
This needs to be adapted for RTL sites:

<img width="202" height="90" alt="Image" src="https://github.com/user-attachments/assets/03043dd3-272a-46a5-a890-21e1438c6a56" />

Why do we use a `:before` pseudo element instead of a border? 🤔